repos/system_upgrade/common/actors/checktargetiso/libraries/check_target_iso.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+from leapp import reporting
+from leapp.libraries.common.config import version
+from leapp.libraries.stdlib import api, CalledProcessError, run
+from leapp.exceptions import StopActorExecutionError
+from leapp.models import TargetOSInstallationImage
+
+
+def inhibit_if_not_valid_iso_file(iso):
+    inhibit_reason = None
+    if not os.path.exists(iso.path):
+        inhibit_reason = 'Provided ISO {} does not exists.'.format(iso.path)
+    else:
+        try:
+            file_cmd_output = run(['file', '--mime', iso.path])
+            if 'application/x-iso9660-image' not in file_cmd_output['stdout']:
+                inhibit_reason = 'The supplied target OS installation image path does not point to a valid ISO image.'
+
+        except CalledProcessError as err:
+            raise StopActorExecutionError(message='Failed to check whether {0} is an ISO file.'.format(iso.path),
+                                          details={'details': '{}'.format(err)})
+    if inhibit_reason:
+        remediation_hint = ('Check whether the supplied target OS installation path points to a valid'
+                            'RHEL{0} ISO image.'.format(version.get_target_major_version()))
+
+        reporting.create_report([
+            reporting.Title('Provided target OS installation ISO is not valid.'),
```

Comment:
Let's differentiate the inhibit reason right in the "Title" so users does not need to go to the report to find out they have typo in the filename.

Reply:
Fixed.